### PR TITLE
pacific: doc/security: Wip doc 2022 05 30 backport pacific security 2nd attempt

### PR DESCRIPTION
**Pull request by zdover23**
doc/security: backport security directory

This PR backports the security/ directory from main/ to pacific/

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by zdover23** [comment]
This PR is underway and will be finished on 31 May 2022.

File: doc/rados/configuration/bluestore-config-ref.rst [hunk 1]
```diff
@@ -139,7 +139,7 @@ In older releases, internal level sizes mean that the DB can fully utilize only
 specific partition / LV sizes that correspond to sums of L0, L0+L1, L1+L2,
 etc. sizes, which with default settings means roughly 3 GB, 30 GB, 300 GB, and
 so forth.  Most deployments will not substantially benefit from sizing to
-accomodate L3 and higher, though DB compaction can be facilitated by doubling
+accommodate L3 and higher, though DB compaction can be facilitated by doubling
```
Comment (anthonyeleven) [inline doc/rados/configuration/bluestore-config-ref.rst:142]: :o

Comment (zdover23) [reply doc/rados/configuration/bluestore-config-ref.rst:142]: :o indeed.

Comment (anthonyeleven) [reply doc/rados/configuration/bluestore-config-ref.rst:142]: That mistake was mine about a year and a half ago.  My ignominy is unbounded.

Comment (zdover23) [reply doc/rados/configuration/bluestore-config-ref.rst:142]: This looks like a typo to me, so I think you're acquitted.

Here's ignominy: Until I was 27, I had to look up "recommend" to see whether it had one or two ms in it. That's ignominy.

Comment (anthonyeleven) [reply doc/rados/configuration/bluestore-config-ref.rst:142]: _Manoever_ remains my nemesis.

**Review by anthonyeleven — APPROVED** [review]

File: doc/rados/configuration/bluestore-config-ref.rst (hunk 1 above)
Comment (zdover23) [reply doc/rados/configuration/bluestore-config-ref.rst:142]: I read this as "Maneater". Like Hall and Oates.

**Comment by zdover23** [comment]
This was part of a wrongheaded attempt to backport release notes. It can be closed without causing problems.

**Pull request closed without merging**